src/IncludedCommands/UsefulFun/fly.lua

```diff
@@ -247,42 +267,101 @@ return {
                         for _,Event in pairs(self.Events) do
                             Event:Disconnect()
                         end
+
+                        --Unbind the ContextActionService bind
+                        ContextActionService:UnbindAction("toggleFlight")
     
                         --Unstore the flight.
                         if Api.CommandData.CurrentFlight == self then
                             Api.CommandData.CurrentFlight = nil
                         end
                     end
+
+                    ContextActionService:BindAction("toggleFlight",function(actionName, UserInputState, InputObject)
+                        -- Prevent multiple fires on 1 press
+                        if UserInputState ~= Enum.UserInputState.End then return end
+                        --Toggle the flight.
+                        if Flight.Active then
+                            (Flight :: any):Stop()
+                        else
+                            (Flight :: any):Start()
+                        end
+                    end,true,Enum.KeyCode.E,Enum.KeyCode.ButtonX)
+                    ContextActionService:SetTitle("toggleFlight","Toggle Flight")
+                    ContextActionService:SetPosition("toggleFlight",UDim2.new(0,50,0,100))
```

Comment:
These 3 lines should have spaces after the commas. Requested lines:
```suggestion
                    end, true, Enum.KeyCode.E, Enum.KeyCode.ButtonX)
                    ContextActionService:SetTitle("ToggleFlight", "Toggle Flight")
                    ContextActionService:SetPosition("ToggleFlight", UDim2.new(0, 50, 0, 100))
```